docker-compose.yml

```diff
@@ -42,6 +81,22 @@ services:
         condition: service_completed_successfully
     expose:
       - "${RPCPORT}"
+    healthcheck:
+      test: ["CMD", "python", "backend/healthcheck.py", "--port", "${RPCPORT}"]
+      interval: 1s
+      timeout: 1s
+      retries: 15
+      start_period: 1s
+    restart: always
+    security_opt:
+      - "no-new-privileges=true"
+    logging:
+      driver: "json-file"
+      options:
+        max-size: "10m"
+        max-file: "3"
+    deploy:
+      replicas: ${JSONRPC_REPLICAS:-1}
```

Comment:
this needs to be always 1

Reply:
It is set to 1 replica by default, could be scaled up using this env var

Reply:
Remove this, it's not being used and doesn't make sense